tests/topotests/bgp-path-attributes-topo1/test_bgp_path_attributes.py

```diff
@@ -500,24 +547,25 @@ def test_localpref_attribute(request):
             }
         }
     }
-
     result = create_router_bgp(tgen, topo, input_dict_4)
     assert result is True, "Testcase {} : Failed \n Error: {}".format(
         tc_name, result)
 
     # Verifying best path
     dut = "r1"
-    attribute = "localpref"
-    result = verify_best_path_as_per_bgp_attribute(tgen, "ipv4", dut,
-                                                   input_dict, attribute)
-    assert result is True, "Testcase {} : Failed \n Error: {}".format(
-        tc_name, result)
+    attribute = "aspath"
+    for addr_type in ADDR_TYPES:
+        result = verify_best_path_as_per_bgp_attribute(tgen, addr_type, dut,
+                                                   {"r7": input_dict["r7"]},
+                                                   attribute)
+        assert result is True, "Testcase {} : Failed \n Error: {}".format(
+            tc_name, result)
 
     write_test_footer(tc_name)
 
 
-def test_weight_attribute(request):
-    " Verifying WEIGHT attribute functionality"
+def test_localpref_attribute(request):
+    " Verifying LOCAL PREFERENCE attribute functionality"
```

Comment:
nit: LOCAL_PREF would be better in cases when match/replace something by pattern

Reply:
Its been taken care. LOCAL PREFERENCE is used only in doc string. Other places it is local_pref/localpref.
Thanks,